nlp/spacy_3_projects/README.md

```diff
@@ -0,0 +1,88 @@
+
+If you're new to Spacy or NER training using Spacy, we definitely recommend having a look at the Spacy docs, or checking out this awesome [Intro to NLP video](https://www.youtube.com/watch?v=IqOJU1-_Fi0) on the Explosion youtube channel. 
+
+## 🏋️ Training a German NER model 
+Since the pre-trained German transformer pipeline does not contain a NER component anymore, we've created a spaCy project that can be used to train a German NER model on a combination of 2 datasets (germeval and wikiner).  
+
+Since we're only going to use training components that are already available in spaCy, we don't need to write actual Python code to run the training! All we need to do is make sure the data is in the correct format (using the `data_*.py` scripts) and run the spaCy train CLI command with the desired config.  
+We've provided 3 different config files:  
+
+1. cpu_eff.cfg: Runs fast on CPU at the cost of lower accuracy.
+2. cpu_acc.cfg: Runs slower on CPU but has a better accuracy.
+3. gpu_trf.cfg: Runs on GPU (or very slowly on CPU) and has the best accuracy.  
+
+In order to switch between them, all you need to do is change the `config` variable in the `project.yml` file and update the `gpu` variable to -1 for cpu and 0 for gpu. If you're running on GPU, also make sure to install the [correct extra dependency](https://spacy.io/usage#gpu).  
+Finally, we've also provided two commands (implemented in the `visualize_*.py` scipts) to visualize the input data and the trained model using [Displacy](https://spacy.io/usage/visualizers#ent) and [Streamlit](https://www.streamlit.io/).
+
+### ⚙️ Running the project end-to-end
+Running everything end to end is pretty easy:
+
+1. Run `spacy project assets`. This will fetch the necessary assets (wikiner and germeval datasets) from the locations provided in the project definition.
+2. Run `spacy project run all`. This will run all the steps defined in the `all` workflow (i.e. corpus, train, evaluate). It will prepare the datasets and combine them, run the training using the config you selected, evaluate the trained model on the test set.
+3. Run `spacy project run visualize-model`. This will launch a streamlit app to visualize the model outputs on custom input you provide.
+4. Run `spacy project run package`. This will package your model so it can be installed and reused in the future.   
+
+Make sure to check out the excellent documentation if you want to learn more about [spaCy projects](https://spacy.io/usage/projects) and the project.yml structure or the structure of the [training configuration](https://spacy.io/api/data-formats) files.
+
+The following part of the readme explains what commands are available in the spaCy project and how you can run them. Btw, the rest of this readme was automatically generated by spaCy, based on the project.yml 😲!
+
+<!-- SPACY PROJECT: AUTO-GENERATED DOCS START (do not remove) -->
+
+# 🪐 spaCy Project: German Named Entitity Recognition
```

Comment:
This emoji is not rendering (it's just a square). Although it is automatically generated, maybe we can change it to something that does work?

Reply:
It works for me

Reply:
![Screenshot 2021-02-02 at 11 22 32](https://user-images.githubusercontent.com/11432563/106586602-f658d680-6548-11eb-843f-b854c0b608e2.png)

Reply:
Weird, works for me as well. Might leave it out though, just to avoid potential problems with some people